mmdet/datasets/coco.py

```diff
@@ -383,19 +383,20 @@ def format_results(self, results, jsonfile_prefix=None, **kwargs):
         result_files = self.results2json(results, jsonfile_prefix)
         return result_files, tmp_dir
 
-    def evaluate(self,
-                 results,
-                 metric='bbox',
-                 logger=None,
-                 jsonfile_prefix=None,
-                 classwise=False,
-                 proposal_nums=(100, 300, 1000),
-                 iou_thrs=None,
-                 metric_items=None):
-        """Evaluation in COCO protocol.
-
+    def evaluate_ins_json(self,
+                          results,
+                          result_files,
+                          metrics,
+                          logger=None,
+                          classwise=False,
+                          proposal_nums=(100, 300, 1000),
+                          iou_thrs=None,
+                          metric_items=None):
+        """Instance segmentation evaluation in COCO protocol.
```

Comment:
Where is bbox evaluation now？

Reply:
`evaluate_ins_json` can do evaluation for instance segmentation and detection. And I will update docstring.